Tests/feaLib/data/spec8c.ttx

```diff
@@ -39,7 +39,7 @@
         <Feature>
           <FeatureParamsStylisticSet>
             <Version value="0"/>
-            <UINameID value="256"/>  <!-- Feature description for MS Platform, script Unicode, language English -->
+            <UINameID value="256"/>  <!-- Feature description for Apple Platform, script Roman, language unspecified -->
```

Comment:
so I'm curious about how the comment strings are determined. Is it merely the text of the first entry with this ID?

Reply:
for the comments we use name table's `getDebugName` method:

https://github.com/fonttools/fonttools/blob/164d6ead033c506c19647778c3e739f029b0ad5f/Lib/fontTools/ttLib/tables/_n_a_m_e.py#L102-L121

Reply:
so yeah, if Macintosh English (platformID=1 langID=0) names exist that match the requested nameID, we return those since they come before Windows English names (platformID=3 langID=0x409)

Reply:
We probably change that code to drop platform and script part if multiple entries of the same NamID exist as it is just misleading. Or may be drop it unconditionally, does not sound like much useful information.